sql/database.sql

```diff
@@ -1526,22 +1526,40 @@ CREATE  TABLE `erx_ttl_touch` (
 -----------------------------------------------------------
 
 --
--- Table structure for table `erx_drug_paid`
---
-
-DROP TABLE IF EXISTS `erx_drug_paid`;
```

Comment:
The table "erx_drug_paid" is still referenced in these files:

```
/library/ajax/drug_autocomplete/search.php
/library/classes/Prescription.class.php
/library/weno/src/AdminProperties.php
```

Reply:
here is the repo for the changes https://github.com/openemr/openemr/pull/2450